stig/xml/article_stig.xml

```diff
@@ -35,6 +36,32 @@
         .
       </para>
     </abstract>
+    <meta name="title" its:translate="yes">Hardening &productname; with &stiga;</meta>
+    <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+    <meta name="description" its:translate="yes">How to audit and harden &productname; with &stig; by the &disa;</meta>
+    <meta name="social-descr" its:translate="yes">Harden &productnameshort; with &stiga;;</meta>
+    <meta name="task" its:translate="yes">
+      <phrase>Auditing</phrase>
+      <phrase>Compliance</phrase>
+    </meta>
+    <revhistory xml:id="rh-article-stig">
```

Comment:
@sounix000 : Please check if the revhistory makes sense.

Reply:
Yes, the revision history is correct. Must have been a painfully boring task for you to dig through the commit and merge messages. Thanks for doing it; I will keep this in mind when I do PCI-DSS and HIPPA.